### DISCUSSION TITLE
Restrict visibility on "Home-button"

### DISCUSSION THREAD

**Discussion by rinjiboy** [pimcore/pimcore#13067] [Q&A / Support]
Hi, 
We would like to restrict functionality and visibility on the Home-Button for specific users/roles. 
![image](https://user-images.githubusercontent.com/43405833/188435932-49ca600c-edea-46f6-8ffd-0c768b0fce99.png)
Is this even possible? 
If so, how can this be done, the easiest way? For now we tried to extend the AdminController, but the system doesn't even address the extending Controller.

**Comment by NiklasBr** [comment]
Check out [Custom Views](https://pimcore.com/docs/pimcore/current/Development_Documentation/Objects/Object_Classes/Class_Settings/Custom_Views.html) and [Perspectives](https://pimcore.com/docs/pimcore/current/Development_Documentation/Tools_and_Features/Perspectives.html). These can then be combined with [Role Permissions](https://pimcore.com/docs/pimcore/current/Development_Documentation/Administration_of_Pimcore/Users_and_Roles.html#page_System-Permissions) where you assign one or more perspectives to a Role.

**Reply by rinjiboy** [reply]
That helped me a little bit, thanks. But the problem is, when the user is logged in, he can only see articles with a specific status. This rule does not apply anymore in the GridView. 

So we try to keep the green box and only want to get rid of the red box. 
![image](https://user-images.githubusercontent.com/43405833/188483534-e0c9321e-033e-4995-af0e-e1ba64633d2a.png)

is that also possible?

**Reply by rinjiboy** [reply]
Setting showroot to false would not get the job done, since too much functionality would get lost.

**Reply by rinjiboy** [reply]
Do you have another tipp by any chance?
